spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -56,7 +56,7 @@ def expect_equal output, expected
         output = builder.text_field :name
 
         expect_equal output, [
-          '<div class="form-group error">',
+          '<div class="form-group error" id="error_person_name">',
           '<label id="error_person_name" class="form-label" for="person_name">',
```

Comment:
The same `id` value is on both the `div` and the `label`. HTML5 spec says an `id` value [must be unique](https://www.w3.org/TR/html5/dom.html#the-id-attribute).

We could either
- change `id` on the label, e.g. `error_label_person_name`

or
- remove the label `id`

I prefer keeping an id on the label, as the GOV.UK elements guide has an [id on the label](https://govuk-elements.herokuapp.com/errors/#highlight-errors). What do you think?